purchase_merge/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_purchase_merge_automatic_wizard_user,access_purchase_merge_automatic_wizard_user,model_access_purchase_merge_automatic_wizard,base.group_user,1,1,1,1
```

Comment:
@BojanOD 

Any reason to use base.group_user instead Purchase User group ?

Reply:
For security reason, I think it's better. Even if a user without purchasing rights is prevented from seeing the wizard menu (merge), it is better to add the purchasing group..